packages/integration/lib/src/views/compose/compose_screen.dart

```diff
@@ -188,7 +194,12 @@ class _LMFeedComposeScreenState extends State<LMFeedComposeScreen> {
             source: LMFeedWidgetSource.createPostScreen,
             backgroundColor: feedTheme.container,
             bottomSheet: _defMediaPicker(),
-            appBar: widget.composeAppBarBuilder?.call(_defAppBar()) ??
+            appBar: widget.composeAppBarBuilder?.call(
+                    _defAppBar(),
+                    onPostCreate,
+                    _defPostCreateButton(),
+                    _defCancelButton(),
+                    onPostValidationFailed) ??
                 widgetUtility.composeScreenAppBar(context, _defAppBar()),
```

Comment:
shouldn't have we add all the parameters in `widgetUtility.composeScreenAppBar` for better control over customisation.

Reply:
Sure